.github/workflows/ci-mac.yml

```diff
@@ -82,6 +82,7 @@ jobs:
     - name: Run tests
       env:
         PLOOMBER_STATS_ENABLED: false
+        CI_CLOUD_API_KEY: ${{ secrets.CI_CLOUD_API_KEY }}
```

Comment:
I think this should have a more specific name, maybe `PLOOMBER_CLOUD_API_KEY`? and looks like this is also missing in the cloud docs

Reply:
This is the key that gets from git secrets to run the CI why does it needs to be in the docs?

Reply:
because I see value in having this for end-users as well, give them the option to use and env variable to set the key

Reply:
I hear you, we can change the naming here, having an env var is another feature regardless to this testing var.